src/InfobipSmsService.php

```diff
@@ -29,11 +32,15 @@ class InfobipSmsService
      * @param $username
      * @param $password
      */
-    public function __construct($from, $username, $password)
+    public function __construct($from, $username, $password, $apiKey = null)
```

Comment:
```suggestion
    * @param  string|null  $apiKey
    public function __construct($from, $username, $password, $apiKey = null)
```
@mattalah please update docsblock here